models/Source/SP/DP_Ph1_Transformer.cpp

```diff
@@ -0,0 +1,272 @@
+/* Copyright 2017-2021 Institute for Automation of Complex Power Systems,
```

Comment:
@JTS22 This was a mistake I guess? I need to check if these files were deleted later.

Reply:
Yes, these files should not be there. I deleted them before but I think they got re-added during the rebase. I just added a new commit to this PR to remove them.